tests/TestCase/Routing/Route/RedirectRouteTest.php

```diff
@@ -236,4 +236,17 @@ public function testParsePersistMatchesAnotherRoute()
         $route = new RedirectRoute('/:lang/my_controllers', ['controller' => 'tags', 'action' => 'add'], ['lang' => '(nl|en)', 'persist' => ['lang']]);
         $route->parse('/nl/my_controllers/');
     }
+
+    /**
+     * Test setting HTTP status
+     *
+     * @return void
+     */
+    public function testSetStatus()
+    {
+        $route = new RedirectRoute('/home', ['controller' => 'posts']);
+        $result = $route->setStatus(302);
+        $this->assertSame($result, $route, 'Should return this');
```

Comment:
"Should return a 302 status code" or no message.

Reply:
Thanks, I removed the messages. Because it checks if the whole Route object is the same.